app/server/locale/ussd.en.yml

```diff
@@ -53,17 +59,24 @@ en:
       2. Cancel
       0. Back
     directory_listing: |
-      CON Choose Market Category
-      1. Food
-      2. Water
-      3. Energy
-      4. Education
-      5. Health
-      6. General shop
-      7. Environment
-      8. Transport
-      9. Labour
+      CON Choose Market Category 
+      %{options}
+      9. Show other options
     complete: END Your request has been sent. You will receive an SMS shortly.
+    directory_listing_other:
+      first: |
+        CON Choose Market Category 
+        %{other_options}
+        9. Show other options
+      middle: |
+        CON Choose Market Category 
+        %{other_options}
+        9. Show other options
+        10. Show previous options
+      last: |
+        CON Choose Market Category 
+        %{other_options}
+        10. Show previous options
```

Comment:
would this show 1-8 and then skip to 10? what would happen if they pressed 9?

Reply:
Yes. I thought it would make more sense to leave that one out so 9 and 10 have the same meaning in both menus.

If you enter 9 you stay in the same menu. Due to this:
            current_menu_nr = current_menu_nr + 1 if current_menu_nr < max_menu else max_menu
https://github.com/teamsempo/SempoBlockchain/blob/b3ba6d0203898313730f0f94aab2a622d8dc28c6/app/server/utils/ussd/kenya_ussd_state_machine.py#L193

Reply:
agreed on the UX, and sounds like mistakes won't cause issues which is great!